source/features/last-notification-page-button.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import looseParseInt from '../helpers/loose-parse-int';
+
+function init(): void {
+	const nextButtonURL = new URL(select('a[data-hotkey="ArrowRight"]')!.href);
+	const nextButtonSearch = new URLSearchParams(nextButtonURL.search);
+	const lastNotificationPageNode = select('.js-notifications-list-paginator-counts')!.lastChild!;
+	const lastNotificationPageNumber = looseParseInt(lastNotificationPageNode);
+	const lastCursor = Math.floor(lastNotificationPageNumber / 50) * 50;
+	nextButtonSearch.set('after', btoa('cursor:' + String(lastCursor)));
+	nextButtonURL.search = nextButtonSearch.toString();
+	lastNotificationPageNode.replaceWith(<> of <a style={{textDecoration: 'underline'}} className="rgh-last-notification-page-button" href={nextButtonURL.href}>{lastNotificationPageNumber}</a></>);
```

Comment:
I think this API accepts multiple parameters so you can wrap this properly, without `<>`

```js
replaceWith(
  " of ",
  <a
    className
    etc
  >
    {number}
  </a>